src/plugins/intel_cpu/src/nodes/mkldnn_experimental_detectron_generate_proposals_single_image_node.cpp

```diff
@@ -103,18 +106,19 @@ void refine_anchors(const float* deltas, const float* scores, const float* ancho
             proposals[p_idx + 1] = y0;
             proposals[p_idx + 2] = x1;
             proposals[p_idx + 3] = y1;
-            proposals[p_idx + 4] = (min_box_W <= box_w) * (min_box_H <= box_h) * score;
+            proposals[p_idx + 4] = score;
+            proposals[p_idx + 5] = (min_box_W <= box_w) * (min_box_H <= box_h) * 1.0;
```

Comment:
We should stay this file as unchanged. But we need an additional file, mkldnn_generate_proposals.cpp/hpp for operation proposed by you.